src/ert/gui/ertwidgets/analysismodulevariablespanel.py

```diff
@@ -135,7 +147,7 @@ def update_inversion_algorithm(self, text: str) -> None:
         self.truncation_spinner.setEnabled(
             not any(val in text.lower() for val in ["direct", "exact"])
         )
-        self.analysis_module.inversion = text
+        self.analysis_module.inversion = text  # type: ignore[assignment]
```

Comment:
(nitpick) Would consider just adding a setter to AnalysisModule.inversion if it doesn't bloat the code too much, it is a BaseModel and pydantic2 supports it